packages/lib/src/components/internal/SecuredFields/lib/utilities/dom.ts

```diff
@@ -62,4 +62,14 @@ const removeAllChildren = node => {
     }
 };
 
-export { getAttribute, on, off, select, selectOne, removeAllChildren };
+const getSearchParameters = (search = window.location.search) =>
+    search
+        .replace(/\?/g, '')
+        .split('&')
+        .reduce((acc, cur) => {
+            const [key, prop = ''] = cur.split('=');
+            acc[key] = decodeURIComponent(prop);
+            return acc;
+        }, {}) as any;
```

Comment:
I think this method is not used anymore, right? If that's the case, can we remove it?